Formula/mk-configure.rb

```diff
@@ -0,0 +1,22 @@
+class MkConfigure < Formula
+  desc "Lightweight replacement for GNU autotools"
+  homepage "https://github.com/cheusov/mk-configure"
+  url "https://downloads.sourceforge.net/project/mk-configure/mk-configure/mk-configure-0.29.3/mk-configure-0.29.3.tar.gz"
+  sha256 "1f483f717a09d06c82a6f3f284dd34234f54ead032e6987d33f3cf8109490c05"
+
+  depends_on "bmake"
```

Comment:
During run or just during build?

Reply:
bmake is both runtime and build dependency.